Makefile

```diff
@@ -54,3 +54,19 @@ ut-circle: calico_mesos_builder.created dist/calico_isolator
 	--with-xunit --xunit-file=/circle_output/output.xml; RC=$$?;\
 	[[ ! -z "$$COVERALLS_REPO_TOKEN" ]] && coveralls || true; exit $$RC'
 
+rpm:
+	cd packages && docker build -t calico-mesos-rpm-builder .
+	mkdir -p $(WD)/packages/rpms
+	mkdir -p $(WD)/packages/srpms
+	cd packages && docker run --name=calico-mesos-rpm-builder \
+	 -v $(WD)/packages/rpms:/opt/rpms \
```

Comment:
Try volume mounting this to `dist` so the rpm's are placed into an "output" directory instead of a "source" directory. Changing it to that is actually 50% of the work required to get circleci creating builds :)